sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1083,7 +1103,7 @@ updated inside the executable graph to reflect the current values in `node`.
 
 Updating these values will not change the structure of the graph.
 
-The implementation may perform a blocking wait during this call on any
+Note: The implementation may perform a blocking wait during this call on any
```

Comment:
I'm also wondering if this should be "The implementation **will** perform a blocking wait during this call on any in-flight executions of that same graph if required by the backend." rather than "may", if we're expecting user code to rely on it. Thoughts @Bensuo

Reply:
Functionally I feel like they both mean the same thing. The user doesn't have a way to reason about whether the backend will actually require a wait, so I feel like "may" better conveys that it is optional. It is quite a subtle distinction though so I don't feel that strongly about it.

Either way the call is safe for the user - they never have to call `wait()` themselves before updating.

Reply:
We are already documenting that the user doesn't have to wait for in-flight submissions in here:

"Updates to a graph will be scheduled after any in-flight executions of the same
graph and will not affect previous submissions of the same graph. The user is
not required to wait on any previous submissions of a graph before updating it."

So I think it doesn't matter much in this case. Don't mind changing the wording if that's what we prefer.

Reply:
Cool, lets just remove the "Note:" prefix for consistency with the other update() API wording, and forget about my wording nitpicks.